javascript/1846-maximum-element-after-decreasing-and-rearranging.js

```diff
@@ -0,0 +1,26 @@
+
+/**
+ * Sorting
+ * Time O(n*log(n)) | Space O(n)
+ * https://leetcode.com/problems/maximum-element-after-decreasing-and-rearranging/
+ * @param {number[]} arr
+ * @return {number}
+ */
+var maximumElementAfterDecrementingAndRearranging = function(arr) {
+    
+    arr.sort((a,b) => a-b);
```

Comment:
For line 11, add a space between a and b to improve readability, see suggestion below
```suggestion
    arr.sort((a, b) => a - b);
```